fighters/jack/src/acmd/smashes.rs

```diff
@@ -61,10 +59,10 @@ unsafe fn jack_attack_s4_s_game(fighter: &mut L2CAgentBase) {
         }
     }
     frame(lua_state, 19.0);
+    FT_MOTION_RATE(fighter, 1.111);
```

Comment:
lets change this to the standard `FT_MOTION_RATE` format `FT_MOTION_RATE(fighter, C / (B - A))`